api/src/controllers/search.js

```diff
@@ -798,6 +798,17 @@ const executeQuery = async function (args, res, next) {
 
   if (dataset[0] === 'Item') {
     defaultLog.info('ITEM GET', { _id: args.swagger.params._id.value });
+    // It's possible to bring down the API if you pass in an invalid item ID, or null.
+    // a post to "api/search?dataset=Item&_id=some-non-object-id" will cause the server to die.
+    // This will still let any 12 char value through (it passes isValid)
+    if (!args.swagger.params._id || !ObjectID.isValid(args.swagger.params._id)) {
```

Comment:
Should be `args.swagger.params._id.value` instead of `args.swagger.params._id` on this line.

Reply:
good catch 👍